### PR TITLE
Add New Auto Server Create Test Script

### PR DESCRIPTION
**Pull request by zhichengzhang1995**
Add New Auto Server Create Test Script

### CODE REVIEW

File: test/Auto_test/ptest.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+source ./shared-queens.rc
+openstack token issue
```
Comment (zhaoqin-github) [inline test/Auto_test/ptest.sh]: Need to install openstack command fist.

File: test/Auto_test/ptest.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+done
+
+while true; do 
+	echo "sshing $ip..."
+	ssh -o "StrictHostKeyChecking no" -i zz.pem ubuntu@$ip echo x
+	if [ $? -eq 0 ]; then break; fi;
+	sleep 1;
+done
+
+ssh -o "StrictHostKeyChecking no" -i zz.pem ubuntu@$ip
```
Comment (zhaoqin-github) [inline test/Auto_test/ptest.sh]: Should not do this in Travis CI.

File: test/Auto_test/ptest.sh [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+done
+echo $p
+
+ip=`openstack server list --name test_instance -f value -c Networks | cut -c13-`
+# get ip
+
+echo $ip
+
+pass=`ping $ip -c 1 -t 2 | awk 'NR==5{print}' | cut -c24-25`
+while [ [$pass == " "] ]
```
Comment (zhaoqin-github) [inline test/Auto_test/ptest.sh]: Need to exit this loop after a lot of attempts fail.

File: test/Auto_test/ptest.sh [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    p=`openstack server list --name test_instance -f value -c Status`
+done
+echo $p
+
+ip=`openstack server list --name test_instance -f value -c Networks | cut -c13-`
+# get ip
+
+echo $ip
+
+pass=`ping $ip -c 1 -t 2 | awk 'NR==5{print}' | cut -c24-25`
```
Comment (zhaoqin-github) [inline test/Auto_test/ptest.sh]: Is it possible to check return code, so that we can make it a little simpler?

File: test/Auto_test/ptest.sh [hunk 5]
```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+source ./shared-queens.rc
+openstack token issue
+# get token
```
Comment (zhaoqin-github) [inline test/Auto_test/ptest.sh]: You needn't to issue a token at first, right?

Comment (zhichengzhang1995) [reply test/Auto_test/ptest.sh]: Yes, I have fixed it.

**Pull request closed without merging**